src/sqlfluff/dialects/dialect_postgres.py

```diff
@@ -1001,15 +1003,9 @@ class IntoClauseSegment(BaseSegment):
 
 
 @postgres_dialect.segment(replace=True)
-class UnorderedSelectStatementSegment(BaseSegment):
+class UnorderedSelectStatementSegment(ansi.UnorderedSelectStatementSegment):
```

Comment:
By using actual Python class inheritance, we eliminate a lot of duplicate code from the base class.